File: app/src/main/java/com/geridea/trentastico/network/request/listener/LessonWithRoomFetchedListener.java
package com.geridea.trentastico.network.request.listener;


/*
 * Created with ♥ by Slava on 13/04/2017.
 */

import com.geridea.trentastico.model.LessonSchedule;

public interface LessonWithRoomFetchedListener {
    void onUpdateSuccessful(LessonSchedule lesson);
    void onError(LessonSchedule lesson);
}

File: app/src/main/java/com/geridea/trentastico/network/controllers/listener/LessonWithRoomFetchedListener.kt
package com.geridea.trentastico.network.controllers.listener


/*
 * Created with ♥ by Slava on 13/04/2017.
 */

import com.geridea.trentastico.model.LessonSchedule

interface LessonWithRoomFetchedListener {
    fun onUpdateSuccessful(lesson: LessonSchedule)
    fun onError(lesson: LessonSchedule)
}
